src/test/java/it/gov/pagopa/payhub/activities/activity/SendEmailIngestionFlowActivityTest.java

```diff
@@ -0,0 +1,194 @@
+package it.gov.pagopa.payhub.activities.activity;
+
+import it.gov.pagopa.payhub.activities.activity.utility.SendEmailIngestionFlowActivity;
+import it.gov.pagopa.payhub.activities.activity.utility.SendEmailIngestionFlowActivityImpl;
+import it.gov.pagopa.payhub.activities.config.EmailTemplatesConfiguration;
+import it.gov.pagopa.payhub.activities.dao.IngestionFlowFileDao;
+import it.gov.pagopa.payhub.activities.dto.*;
+import it.gov.pagopa.payhub.activities.service.SendMailService;
+import it.gov.pagopa.payhub.activities.service.UserAuthorizationService;
+import jakarta.mail.MessagingException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.TestPropertySource;
+
+import java.util.Optional;
+
+import static org.mockito.ArgumentMatchers.any;
+
+/**
+ * test class for send mail for ingestion activity
+ */
+@SpringBootTest(
+  classes = {
+    SendEmailIngestionFlowActivityImpl.class,
+    EmailTemplatesConfiguration.class
+  },
+  webEnvironment = SpringBootTest.WebEnvironment.NONE)
+@TestPropertySource(properties = {
+    "activity.root.path=/DATA/"
+})
```

Comment:
this should not be needed if present on the application.yml

Reply:
OK, removed